package.json

```diff
@@ -11,12 +13,16 @@
     "nyc": "^11.4.1"
   },
   "scripts": {
-    "test": "etc/prepare-tests.sh && nyc mocha **/*.spec.js",
-    "test-report": "etc/prepare-tests.sh && nyc mocha -R xunit **/*.spec.js > build/test-reports/synctos.xml"
+    "test": "etc/prepare-tests.sh && nyc mocha \"**/*.spec.js\"",
+    "test-report": "etc/prepare-tests.sh && nyc mocha -R xunit \"**/*.spec.js\" > build/test-reports/synctos.xml"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/Kashoo/synctos"
+  },
+  "bin": {
+    "synctos": "./make-sync-function",
+    "synctos-validate": "./validate-document-definitions"
```

Comment:
These entries ensure that a project that depends on synctos will automatically install scripts with the specified names into the dependent project's `node_modules/.bin` directory.

Reply:
slick az